Article/Article.js

```diff
@@ -3,27 +3,39 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'expand';
+    this.read = document.createElement('button');
+    this.read.textContent = 'read';
+    this.domElement.appendChild(this.read);
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
+    this.read.addEventListener('click', () => this.articleRead());
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle('article-open');
+    this.domElement.classList.contains('article-open') ? this.expandButton.textContent = 'close' : this.expandButton.textContent = 'expand';
```

Comment:
Good job making this button dynamic! It adds lots of feedback for the user and can make things more intuitive for them.